src/binaries/query/main.rs

```diff
@@ -73,6 +76,17 @@ async fn main(_global_tracker: Arc<RuntimeTracker>) -> common_exception::Result<
     init_default_metrics_recorder();
     set_panic_hook();
 
+    let limits = get_own_limits().unwrap();
```

Comment:
Does this require root permissions??

Reply:
I refer to https://crates.io/crates/limits-rs, but it does not any description of root permission.